mappings/net/minecraft/registry/Registries.mapping

```diff
@@ -69,6 +69,7 @@ CLASS net/minecraft/class_7923 net/minecraft/registry/Registries
 	FIELD field_41195 VILLAGER_PROFESSION Lnet/minecraft/class_7922;
 	FIELD field_42940 DECORATED_POT_PATTERNS Lnet/minecraft/class_2378;
 	FIELD field_43381 RULE_BLOCK_ENTITY_MODIFIER Lnet/minecraft/class_2378;
+	FIELD field_44687 ITEM_GROUPS Lnet/minecraft/class_2378;
```

Comment:
```suggestion
	FIELD field_44687 ITEM_GROUP Lnet/minecraft/class_2378;
```
Should match the key's name, and be single not plural

Reply:
Opps, my bad: fixed here https://github.com/FabricMC/yarn/pull/3549